apps/web/pages/api/book/event.ts

```diff
@@ -637,21 +713,23 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   );
   await Promise.all(promises);
 
-  await prisma.booking.update({
-    where: {
-      uid: booking.uid,
-    },
-    data: {
-      references: {
-        createMany: {
-          data: referencesToCreate,
+  if (booking) {
+    await prisma.booking.update({
+      where: {
+        uid: booking.uid,
+      },
+      data: {
+        references: {
+          createMany: {
+            data: referencesToCreate,
+          },
         },
       },
-    },
-  });
-
-  // booking successful
-  return res.status(201).json(booking);
+    });
+    // booking successful
+    return res.status(201).json(booking);
+  }
+  return res.status(500).json({ message: "There is not a stripe_Payment credential" });
```

Comment:
What do you think if we update `stripe_Payment` to `stripe_payment` to be more consistent since the type is like this?